dbml/gen_model.go

```diff
@@ -8,27 +8,41 @@ import (
 )
 
 type ModelGen struct {
-	Pkg    string
-	Cls    string
-	Models []*ModelInfo
-	Funcs  []*FuncInfo
+	Pkg          string
+	Cls          string
+	DataPkg      string
+	ErrorPkg     string
+	ErrorPkgName string
+	Models       []*ModelInfo
+	Funcs        []*FuncInfo
 }
 
-func NewModelGen(dbml *DBML, pkg string) *ModelGen {
+func NewModelGen(dbml *DBML, pkg, datapkg, dbName, errorPkg string) *ModelGen {
```

Comment:
`datapkg, errorPkg string` perfer changed to `customImports []string`

Reply:
The reason I split these two pkg into 2 fields instead of using an array is I need to determinate which one is errorPkg and get the last default variable name for customised error checking logic to call. If using an array, I need to hard code which position refers to what pkg, which I think it's not a good way for readability. Please let me know if using an array is essential for this case :)